Makefile

```diff
@@ -190,8 +190,12 @@ $(CONTROLLER_GEN): $(LOCALBIN)
 	test -s $(LOCALBIN)/controller-gen && $(LOCALBIN)/controller-gen --version | grep -q $(CONTROLLER_TOOLS_VERSION) || \
 	GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-tools/cmd/controller-gen@$(CONTROLLER_TOOLS_VERSION)
 
+# Download CRDs for envtest
+crddir/github.com/open-telemetry/opentelemetry-operator:
+	git clone --depth 1 --branch v0.93.0 https://github.com/open-telemetry/opentelemetry-operator.git crddir/github.com/open-telemetry/opentelemetry-operator
```

Comment:
[nit]
Could you please extract the branch into a variable? Also could you please bump it to .94  so it is in sync with the collector image version?